docs/modules/development/pages/oia/introduction.adoc

```diff
@@ -43,6 +43,34 @@ See the interfaces defined in the `api` module for a complete list.
 ** https://github.com/OpenNMS/opennms-integration-api/blob/master/api/src/main/java/org/opennms/integration/api/v1/timeseries/README.md[Time series storage]
 ** Graph API extensions (i.e., custom topology)
 
-== Start a new project with OIA
+== Start and build a new project with OIA
 
-== Build a new plugin
+OIA provides a Maven archetype to help bootstrap plugin creation.
+
+Create a new plugin project using below maven command :
+
+```
+mvn archetype:generate -B -DarchetypeGroupId=org.opennms.integration.api -DarchetypeArtifactId=example-kar-plugin -DarchetypeVersion=1.0.0-SNAPSHOT -DgroupId=com.company -DartifactId=myplugin -Dversion=0.1.0-SNAPSHOT -Dpackage=com.company.myplugin -DpluginId=myplugin -DpluginName="My Plugin"
+```
+
+NOTE: You can replace name `myplugin` and package name `com.company.myplugin` and version with your own choices. Generated plugin adds some Java files and feature definitions which are only added as sample. They can be updated and deleted when needed.
```

Comment:
NOTE: You can replace name `myplugin`, package name `com.company.myplugin`, and version with your own choices. 
Generated plugin adds some Java files and feature definitions which are only added as samples. 
You can update and delete them when needed.